terraform/outputs.tf

```diff
@@ -0,0 +1,10 @@
+output "app_name" {
+  value = juju_application.metacontroller_operator.name
+}
+
+output "provides" {
+  value = {
+    grafana_dashboard = "grafana-dashboard",
+    metrics_endpoint  = "metrics-endpoint",
+  }
+}
```

Comment:
After discussion with @orfeas-k, we think having the output "requires" still exist and be empty make more sense, like this:
```
output "requires" {
  value = {}
}
```